core-feature-pack/common/src/main/resources/content/bin/systemd/README

```diff
@@ -0,0 +1,80 @@
+= How to configure WildFly as a Systemd service
+
+The following steps describe the process to configure WildFly as a Systemd service.
+You can use the provided wildfly-[standalone,domain].service file as a template for your own Systemd Unit File.
+It can be adjusted to your needs, for example, you can manually change the user that runs the service,
+the location of the WildFly installation, logs, etc.
+Alternatively, you can use the generate_systemd_unit.sh script to automatically generate a new Systemd Unit File
+using this server installation as the WildFly home.
+
+If you want to install WildFly as a Systemd service from scratch as a Systemd service, follow the steps below.
+
+== Install WildFly
+
+    # unzip -d /opt wildfly-34.0.0.Final.zip
+    # ln -s /opt/wildfly-34.0.0.Final /opt/wildfly
+
+== Standalone or Domain mode ?
+
+   # MODE=standalone
+
+== Create a wildfly user and group
+
+    # groupadd -r wildfly
+    # useradd -r -g wildfly -d /opt/wildfly -s /sbin/nologin wildfly
+
+== Configure systemd
+
+    # cd /opt/wildfly/bin/systemd
```

Comment:
To avoid hard coding /opt/wildfly everywhere, I would suggest defining a WILDFLY_HOME=/opt/wildfly at the beginning.

Reply:
Done